opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -424,7 +428,7 @@ namespace {
                            const Opm::SummaryState&    st,
                            const Opm::Schedule&        sched,
                            const std::size_t           simStep,
-                           Array&                      sAcn)
+                           const Array&                sAcn)
```

Comment:
I appreciate it, but I'd prefer that this remain non-`const` here.  The code structure **probably** defeats static analysis, but the `sAcn` object really is being mutated here.  In particular, the return value from `Array::operator[]()` is a view into a mutable range, so when we create the per-condition [`sacn`](https://github.com/OPM/opm-common/blob/cd7e2a2c65138a41db19f509bcba4c73c5389ad7/opm/output/eclipse/AggregateActionxData.cpp#L442) object and assign values to it, we're really changing the underlying `Array` object.

Not marking `sAcn` as `const` is intended to signify that fact, but I would really like a better way of doing it.  I'm hopeful that C++20's `std::span<>` will make these semantics more transparent...

Reply:
done. commit removed.